apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -869,23 +869,23 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_max_awaiting_rel {
         desc {
-            en: """Maximum QoS 2 packets (Client -> Broker) awaiting PUBREL."""
-            zh: """PUBREL (Client -> Broker) 最大等待队列长度。"""
+            en: """For each publisher session, the maximum number of outstanding QoS 2 messages pending on the client to send PUBREL. After reaching this limit, new QoS 2 PUBLISH requests will be rejected with `147(0x93)` until either PUBREL is received or timed out."""
+            zh: """每个发布者的会话中，都存在一个队列来处理客户端发送的 QoS 2 消息。该队列会存储 QoS 2 消息的报文 ID 直到收到客户端的 PUBREL 或超时，达到队列长度的限制后，新的 QoS 2 消息发布会被拒绝，并返回 `147(0x93)` 错误。"""
         }
         label: {
             en: """Max Awaiting PUBREL"""
-            zh: """Max Awaiting PUBREL"""
+            zh: """PUBREL 等待队列长度"""
         }
     }
 
     mqtt_await_rel_timeout {
         desc {
-            en: """The QoS 2 messages (Client -> Broker) will be dropped if awaiting PUBREL timeout."""
-            zh: """PUBREL (Client -> Broker) 最大等待时间，超时则会被丢弃。"""
+            en: """For client to broker QoS 2 message, the time limit for the broker to wait before the `PUBREL` message is received. The wait is aborted after timed out, and receiving a stale `PUBREL` causes a warning level log. Note, the message is delivered to subscribers before entering the wait for PUBREL."""
+            zh: """客户端发布 QoS 2 消息时，服务器等待 `PUBREL` 的最长时延。超过该时长后服务器会放弃等待，如果在这之后收到 PUBREL，服务器将会产生一条告警日志。注意，向订阅客户端转发消息的动作发生在进入等待之前。"""
```

Comment:
We need to clarify that after timeout, the resend of the last publish will be treated as a new QoS2 flow, meaning it will not be exact once delivery.

Reply:
see my change suggestion below.